week03/assignment03_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestRate(t *testing.T) {
+
+	var tests = []struct {
```

Comment:
We will cover table driven tests next week, but good use of them here.

Reply:
cool :)